src/cryptonote_core/service_node_list.cpp

```diff
@@ -896,24 +897,51 @@ namespace service_nodes
       return false;
     }
 
-    const uint64_t min_transfer = get_min_node_contribution(hf_version, staking_requirement, info.total_reserved, info.total_num_locked_contributions());
-    if (stake.transferred < min_transfer)
+    if (hf_version >= cryptonote::network_version_16)
     {
-      LOG_PRINT_L1("Register TX: Contribution transferred: " << stake.transferred << " didn't meet the minimum transfer requirement: " << min_transfer << " on height: " << block_height << " for tx: " << cryptonote::get_transaction_hash(tx));
-      return false;
-    }
+      // In HF16 we start enforcing three things that were always done but weren't actually enforced:
+      // 1. the staked amount in the tx must be a single output.
+      if (stake.locked_contributions.size() != 1)
```

Comment:
This might be enforceable after the HF. I also doubt anyone has tried to do this